opus/application/settings.py

```diff
@@ -379,3 +379,6 @@
 
 # We don't want to have these characters in HTML class or ID for customized tooltips.
 INVALID_CLASS_CHAR = '~!@$%^&*()+=,./;:"?><[]\{}|`# '
+
+# The symbol is appened after product_type to specify the files version
+FILE_VERSION_MODIFIER = '@'
```

Comment:
This doesn't really seem like worth including in settings if we're making it a permanent part of the API. We don't do this with anything else in the API.